lib/copy_gauge_helper.cuh

```diff
@@ -1,157 +1,109 @@
-#include <tune_quda.h>
-
-#include <jitify_helper.cuh>
+#include <tunable_nd.h>
 #include <kernels/copy_gauge.cuh>
 
 namespace quda {
 
   using namespace gauge;
 
-  template <typename FloatOut, typename FloatIn, int length, typename Arg>
-  class CopyGauge : TunableVectorYZ {
-    Arg arg;
+#ifdef FINE_GRAINED_ACCESS
+  constexpr bool fine_grain() { return true; }
+#define GAUGE_FUNCTOR CopyGaugeFineGrained_
```

Comment:
You don't expect me to get excited about that ;-)

Reply:
This is something that can be trivially cleaned up with `if constexpr` to remove these macro definitions.  So until we make Jenkins C++17 only, I'll defer until doing that.